components/_global-navigation.scss

```diff
@@ -57,10 +59,52 @@ $nav-width: 1024px;
 		}
 	}
 
+	&__icon-user {
+		border: solid 2px $color-dark-blue-gray;
+		border-radius: 50%;
+		fill: $color-dark-blue-gray;
+		height: 28px;
+		margin-right: 2px;
+		width: 28px;
+	}
+
+	&__notifications {
+		position: relative;
+	}
+
+	&__notification-counter-wrapper {
+		background-color: $color-tv;
```

Comment:
This should be this one I believe (from `colors.scss`):
```
// ## Fandom Corporate Colors
$color-link: #00b7e0;
```
Or should I define one more color? We used to have all color names defined and then used them for feature colors:
```
$color-aqua-blue: #00b7e0;
$color-tv: $color-aqua-blue;
```
Maybe we should go back to this?